### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by dafuga**
And put the repo in a place where we can hopefully start adding more languages in the near future. :smile:

### CODE REVIEW

File: app/modules/handler/components/Fragment/Transaction/Action.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ class PromptFragmentTransactionAction extends Component<Props> {
           ? (
             <Divider horizontal size="large">
               <Header>
-                Action {index + 1} of {total}
+                {t('handler_transaction_action_fuel_label_one', { index: index + 1, total })}
```
Comment (aaroncox) [inline app/modules/handler/components/Fragment/Transaction/Action.js:42]: These aren't fuel related, so ideally the labels here wouldn't say they are.

File: app/modules/handler/containers/Stage/Review.js [hunk 2]
```diff
@@ -55,13 +59,21 @@ class PromptStageReview extends Component<Props> {
...
           <Header>
-            Actions to Perform
+            {t('handler_containers_stage_review_header')}
             <Header.Subheader>
-              Listed below are the actions to be performed. If you are unsure of what this request does,
-              {' '}
-              <a onClick={onShareLink} style={{ cursor: 'pointer' }}>
-                use a URI link to ask those you trust
-              </a>.
+              <div dangerouslySetInnerHTML={{
```
Comment (aaroncox) [inline app/modules/handler/containers/Stage/Review.js:64]: I'm slightly concerned about this and XSS. I'm trying to figure out if something malicious could make it into the code somewhere and cause problems since we're dangerously setting HTML here.

I'd almost prefer to remove this link all together rather than go this route. There's a share button on the bottom that triggers the same thing.

File: app/modules/main/components/Overview/Table/Header.js [hunk 3]
```diff
@@ -23,24 +24,27 @@ class OverviewTableHeader extends Component<Props> {
         </Table.HeaderCell>
       );
     });
+
+    console.log({propsHeader: this.props})
```
Comment (aaroncox) [inline app/modules/main/components/Overview/Table/Header.js:28]: debugs!

File: app/modules/main/containers/ContentError.js [hunk 4]
```diff
@@ -1,9 +1,14 @@
 // @flow
 import React, { Component } from 'react';
 import { bindActionCreators } from 'redux';
+
 import { connect } from 'react-redux';
+import compose from 'lodash/fp/compose';
+import { withTranslation } from 'react-i18next';
 
 import { Container, Form, Header, Segment } from 'semantic-ui-react';
+import { withRouter } from 'react-router-dom';
```
Comment (aaroncox) [inline app/modules/main/containers/ContentError.js:10]: Why was this included? I don't think this file makes use of the router.

File: app/modules/main/containers/Sections/Account/Overview/Ram.js [hunk 5]
```diff
@@ -50,7 +52,11 @@ class AccountOverviewRam extends Component<Props> {
                       marginTop: '0.35em',
                     }}
                   >
-                    A <strong>size-based</strong> resource an account can use to <strong>store data</strong> within smart contracts.
+                    <div
+                      dangerouslySetInnerHTML={{
```
Comment (aaroncox) [inline app/modules/main/containers/Sections/Account/Overview/Ram.js:56]: Another instance of `dangerouslySetInnerHTML` - was this required, and if so, can we do it another way?

File: app/modules/main/containers/Sections/Account/Overview/Resource.js [hunk 6]
```diff
@@ -56,9 +61,11 @@ class AccountOverviewResource extends Component<Props> {
                           marginTop: '0.35em',
                         }}
                       >
-                        A <strong>time-based</strong> resource an account uses
-                        {' '}
-                        while <strong>performing</strong> smart contract actions.
+                        <div
```
Comment (aaroncox) [inline app/modules/main/containers/Sections/Account/Overview/Resource.js:64]: A few more `dangerouslySetInnerHTML` in this file.

File: app/modules/main/index.js [hunk 7]
```diff
@@ -5,6 +5,11 @@ import { AppContainer } from 'react-hot-loader';
 import Root from './containers/Root';
 import Routes from './routes';
 
+const { withTranslation, translate } = require('react-i18next')
+// import i18n from 'react-i18next';
+console.log({withTranslation});
```
Comment (aaroncox) [inline app/modules/main/index.js:10]: debugs!

File: app/shared/components/Global/Message/App/Error.js [hunk 8]
```diff
@@ -1,13 +1,24 @@
 // @flow
 import React, { Component } from 'react';
-import { withTranslation } from 'react-i18next';
+
+const { withTranslation, translate } = require('react-i18next')
+const i18n = require('react-i18next')
+
+// import i18n from 'react-i18next';
```
Comment (aaroncox) [inline app/shared/components/Global/Message/App/Error.js:7]: some debugging

File: app/shared/components/Tools/KeyGenerator.js [hunk 9]
```diff
@@ -219,11 +221,10 @@ class ToolsKeyGenerator extends Component<Props> {
               </Segment>
 
             </React.Fragment>
-          )
-          : (
```
Comment (aaroncox) [inline app/shared/components/Tools/KeyGenerator.js:223]: I've been intentionally formatting these conditional statements in JSX like this:

```
{(logic === true)
   ? ( if )
   : ( else )
}
```

So you can see the `?` and `:` at the start of the lines. I've noticed you correct them a few times, we should come up with what our standard is for this.

File: app/shared/containers/Global/Fuel/Dropdown.js [hunk 10]
```diff
@@ -84,27 +89,38 @@ class GlobalFuelDropdown extends Component<Props> {
...
-                  <a href="#" onClick={() => this.openLink('https://greymass.com/fuel')}>
-                    https://greymass.com/fuel
-                  </a>
-                </p>
+              <p>
+                {t('fuel_paragraph_one')}
+              </p>
+              <p>
+                <div
+                  dangerouslySetInnerHTML={{
```
Comment (aaroncox) [inline app/shared/containers/Global/Fuel/Dropdown.js:97]: Another `dangerouslySetInnerHTML`

File: app/shared/components/Tools/KeyGenerator.js (hunk 9 above)
Comment (dafuga) [reply app/shared/components/Tools/KeyGenerator.js:223]: Ok I see, that makes sense so I'll try to start doing that instead.
